xwiki-pro-macros-test/xwiki-pro-macros-test-docker/src/test/it/com/xwiki/pro/test/ui/GenericMacrosIT.java

```diff
@@ -76,38 +90,170 @@
+            + "{{/panel}}\n";
+
+    private static final String PAGE_WITH_MICROSOFT_STREAM_CONTENT =
+        "{{msStream url=\"www.youtube.com\" alignment=\"RIGHT\" "
+            + "start=\"01:12:13\" showinfo=\"true\" autoplay=\"true\" width=\"600px\" height=\"400px\"/}}\n"
+            + "\n"
+            + "{{msStream url=\"www.youtube.com\" alignment=\"LEFT\" start=\"03:12:13\" textWrap=\"true\"/}}\n"
+            + "\n"
+            + "{{msStream url=\"www.youtube.com\" alignment=\"CENTER\" showinfo=\"false\" autoplay=\"false\"/}}\n";
 
-    private void registerMacros(){
+    private void registerMacros()
+    {
         RegisterMacro register = new RegisterMacro();
-        register.registerMacro(BASE_XWIKI_MACRO_SPACE,"Button");
-        register.registerMacro(BASE_XWIKI_MACRO_SPACE,"MicrosoftStream");
-        register.registerMacro(BASE_XWIKI_MACRO_SPACE,"Panel");
+        register.registerMacro(BASE_XWIKI_MACRO_SPACE, "Button");
+        register.registerMacro(BASE_XWIKI_MACRO_SPACE, "MicrosoftStream");
+        register.registerMacro(BASE_XWIKI_MACRO_SPACE, "Panel");
         register.registerMacro(BASE_XWIKI_MACRO_SPACE, "Team");
-        register.registerMacro(BASE_XWIKI_MACRO_SPACE,"Taglist");
-
+        register.registerMacro(BASE_XWIKI_MACRO_SPACE, "Taglist");
     }
 
     @BeforeAll
     void setup(TestUtils setup)
     {
         setup.loginAsSuperAdmin();
-        setup.createUser("UserTest", "UserTest", "");
-        setup.createUser("UserTest2", "UserTest", "");
-        setup.createUser("UserTest3", "UserTest", "");
+        setup.createUser("UserTest", "UserTest", "", "company", "xwiki", "phone", "07777777", "email"
+            , "usertest@example.com", "address", "userTestAddress", "comment", "test", "blog", "https://example.com/",
+            "blogfeed", "https://example.com/");
+        setup.createUser("UserTest2", "UserTest", "", "company", "xwiki", "phone", "07777777", "email"
+            , "usertest2@example.com", "address", "userTestAddress2", "comment", "test2");
+        setup.createUser("UserTest3", "UserTest", "", "company", "xwiki", "phone", "07777777", "email"
+            , "usertest3@example.com", "address", "userTestAddress3", "comment", "test3", "blog", "https://example"
+                + ".com/", "blogfeed", "https://example.com/");
 
+        setup.setGlobalRights("XWiki.XWikiAllGroup", "", "comment", true);
+        setup.setGlobalRights("XWiki.XWikiAllGroup", "", "edit", true);
         setup.deletePage(pageWithTeamMacros);
-        registerMacros();
+        setup.deletePage(pageWithButtonMacros);
+        setup.deletePage(pageWithStatusMacros);
+        setup.deletePage(pageWithTagListMacros);
+        setup.deletePage(pageWithExpandMacro);
+        setup.deletePage(pageWithUserProfileMacro);
+        setup.deletePage(pageWithUserListMacro);
+        setup.deletePage(PageWithPanelMacros);
+        setup.deletePage(pageWithTags);
+        setup.deletePage(PageWithMStreamMacros);
```

Comment:
You don't need these
This is executed only once before all tests. When it's executed, no pages were created yet -> no need to delete them

I think you needed these since you reused your instance, but that's only for "testing" purposes. While running `mvn failsafe:integration-test ` for an existing instance I guess you can comment the `createPage` if that's breaking something (you get an error because the page exists already?)